geode-redis/src/main/java/org/apache/geode/redis/internal/statistics/GeodeRedisStats.java

```diff
@@ -72,7 +72,7 @@ public GeodeRedisStats(StatisticsFactory factory, String textId,
 
     type = statisticsTypeFactory
         .createType("RedisStats",
```

Comment:
"RedisStats" - Does this string show-up anywhere user facing?

Reply:
I don't think so

Reply:
Okay.  Someone mentioned perhaps that string shows up in VSD?

Reply:
Perhaps if we changed it to something like:
"statsForServerCompatibleWithRedis" ?